projects/smoketest_max6643/src/main.c

```diff
@@ -0,0 +1,58 @@
+#include <stdbool.h>
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "max6643_fan_controller.h"
+#include "wait.h"
+
+#define FANFAIL_PIN \
+  { .port = GPIO_PORT_B, .pin = 4 }
+#define OVERTEMP_PIN \
+  { .port = GPIO_PORT_B, .pin = 5 }
+#define FAN_PIN \
+  { .port = GPIO_PORT_B, .pin = 3 }
+#define FAN_SPEED_HIGH true
```

Comment:
This controls full speed, not "high speed", so `FAN_FULL_SPEED` might be a better name.